### ISSUE TITLE
Compile Error for  6.2.7.17 for kernel 6.0

### ISSUE THREAD

**Issue by gignac1024** [dm-vdo/kvdo#60]
Hi,
this is compile error by dkms, version from https://copr.fedorainfracloud.org/coprs/rhawalsh/dm-vdo/ installed in fedora 36

DKMS make.log for kvdo-6.2.7.17 for kernel 6.0.5-200.fc36.x86_64 (x86_64)
```
 ...
  CC [M]  /var/lib/dkms/kvdo/6.2.7.17/build/vdo/base/vdoResume.o
  CC [M]  /var/lib/dkms/kvdo/6.2.7.17/build/uds/threadsLinuxKernel.o
  CC [M]  /var/lib/dkms/kvdo/6.2.7.17/build/uds/timeUtils.o
/var/lib/dkms/kvdo/6.2.7.17/build/uds/sysfs.c:96:4: error: ‘struct kobj_type’ has no member named ‘default_attrs’
   96 |   .default_attrs = emptyAttrs,
      |    ^~~~~~~~~~~~~
/var/lib/dkms/kvdo/6.2.7.17/build/uds/sysfs.c:96:20: error: initialization of ‘const struct attribute_group **’ from incompatible pointer type ‘struct attribute **’ [-Werror=incompatible-pointer-types]
   96 |   .default_attrs = emptyAttrs,
      |                    ^~~~~~~~~~
/var/lib/dkms/kvdo/6.2.7.17/build/uds/sysfs.c:96:20: note: (near initialization for ‘emptyObjectType.default_groups’)
/var/lib/dkms/kvdo/6.2.7.17/build/uds/sysfs.c:182:4: error: ‘struct kobj_type’ has no member named ‘default_attrs’
  182 |   .default_attrs = parameterAttrs,
      |    ^~~~~~~~~~~~~
/var/lib/dkms/kvdo/6.2.7.17/build/uds/sysfs.c:182:20: error: initialization of ‘const struct attribute_group **’ from incompatible pointer type ‘struct attribute **’ [-Werror=incompatible-pointer-types]
  182 |   .default_attrs = parameterAttrs,
      |                    ^~~~~~~~~~~~~~
/var/lib/dkms/kvdo/6.2.7.17/build/uds/sysfs.c:182:20: note: (near initialization for ‘parameterObjectType.default_groups’)
cc1: all warnings being treated as errors

```

**Comment by rhawalsh** [comment]
This should be fixed with kmod-kvdo-6.2.7.17-4.fc36.  There was a missing patch that somehow kept creeping into my testing (I guess?).  I just tested on a fresh set of F35 (6.0.5-100.fc35), F36 (6.0.5-200.fc36), and Rawhide (6.1.0-0.rc3.20221102git8f71a2b3f435.29.fc38) systems and I was able to create volumes as intended.

**Comment by gignac1024** [comment]
It works, thanks.
